packages/@vuepress/theme-default/src/node/defaultTheme.ts

```diff
@@ -44,7 +44,11 @@ export const defaultTheme: Theme<DefaultThemeOptions> = ({
     clientAppSetupFiles: path.resolve(__dirname, '../client/clientAppSetup.js'),
 
     // use the relative file path to generate edit link
-    extendsPageData: ({ filePathRelative }) => ({ filePathRelative }),
+    // store title to display navbar and sidebar
+    extendsPageData: ({ filePathRelative, title }) => ({
+      filePathRelative,
+      meta: { title },
```

Comment:
If `extendsPageData` is called multiple time, seems that the `meta` object will be replaced, instead of being merged?

Reply:
What about putting it into extendsPageOptions ？ 🤔 (Just an idea, not fully considered)

Reply:
Nope, plugins may need to read user frontmatter then write something as page meta. 

This is the same as frontmatter, currently frontmatter could be possibly overided if users not speard the original ones.

Actually yesterday I was thing to add a new hook, warnings will be displayed if root property of meta was overided. But users may meet situation where they are trying to override some content.

Reply:
I will speard the original meta to fix this in next commit